src/containers/PipelineResource/PipelineResource.js

```diff
@@ -0,0 +1,187 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  DataTable,
+  DataTableSkeleton,
+  InlineNotification
+} from 'carbon-components-react';
+
+import {
+  getPipelineResource,
+  getPipelineResourcesErrorMessage,
+  isFetchingPipelineResources
+} from '../../reducers';
+
+import { fetchPipelineResource } from '../../actions/pipelineResources';
+
+const {
+  TableContainer,
+  Table,
+  TableHead,
+  TableRow,
+  TableBody,
+  TableCell,
+  TableHeader
```

Comment:
nit: alphabetic

Reply:
ah of course, the lint rule only covers imports 🙁  I'll fix the order here and take another look at tooling for this stuff